au.org.access-nri/model/configuration/ci/2-0-0.json

```
@@ -0,0 +1,85 @@
+{
+    "$id": "https://raw.githubusercontent.com/ACCESS-NRI/schema/main/au.org.access-nri/model/configuration/ci/2-0-0.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Model Configuration CI Testing Configuration",
+    "description": "Settings for Continuous Integration (CI) tests for model configurations repositories",
+    "type": "object",
+    "$defs": {
+        "config": {
+            "type": "object",
+            "properties": {
+                "markers": {
+                    "type": "string",
+                    "description": "A python expression of markers to pass to model-config-tests pytests"
+                },
+                "model-config-tests-version": {
+                    "type": "string",
+                    "description": "A version of the model-config-tests package"
+                },
+                "python-version": {
+                    "type": "string",
+                    "description": "The python version used to create test virtual environment"
```

```suggestion
                    "description": "The python version used to create test virtual environment on Github hosted tests"
```

Probably not needed but wondering if it makes it more clear that this will not effect checks run on NCI?